modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -921,16 +921,16 @@ std::string serToString(const LMDBBackend::LMDBResourceRecord& lrr)
 }
 
 template <>
-std::string serToString(const vector<LMDBBackend::LMDBResourceRecord>& lrrs)
+std::string serializeToBuffer(const vector<LMDBBackend::LMDBResourceRecord>& lrrs)
 {
   std::string ret;
   for (const auto& lrr : lrrs) {
-    ret += serToString(lrr);
+    ret += serializeToBuffer(lrr);
   }
   return ret;
 }
 
-static inline size_t serOneRRFromString(const string_view& str, LMDBBackend::LMDBResourceRecord& lrr)
+static inline size_t deserializeRRFromBuffer(const string_view& str, LMDBBackend::LMDBResourceRecord& lrr)
 {
   uint16_t len;
   memcpy(&len, &str[0], 2);
```

Comment:
I suddenly wonder how this works out endian-wise, but that's not this PR's problem

Reply:
Agreed. We need to be more explicit about these things.